scarborough.md

```diff
@@ -0,0 +1,5 @@
+# Scarborough, ON
+
+### Where to eat
```

Comment:
I haven't lived in Scarberia since the 70's
Lot's has changed and I wonder where you would recommend?